src/components/Lesson/ImprovePage.scss

```diff
@@ -81,25 +38,34 @@
 }
 
 /* Student / Teacher mode */
+
+/* Box shadow inset used to highlight box so it differentiates better from other objects */
 .improvePageBoxStudent {
   composes: improvePageBox;
   background-color: $color-green-light;
+  box-shadow: inset 0 0 85px #89c785;
 }
 
+/* Works as a <hr/> to differentiate text and links */
+.improvePageBoxStudent .improvePageBoxTextRow {
```

Comment:
Jeg har et par forslag generelt til denne scss-filen.

For det første, siden vi bruker css-modules, så trenger vi ikke å prefixe alle classes med `improvePageBox`. Jeg tror det blir mer leselig med bare `.linkRow` og `.font` eller lignende.

For det andre, så mener jeg det ikke burde være nødvendig å ha cascading når man bruker css-modules. Istedet for å ha `.student .textRow`, kunne man ikke heller bare hatt  `.textRow`?

Den siste kommentaren er litt smak og behag, men jeg må innrømme at jeg ofte foretrekker å ikke bruke så mye "composes", men heller bare inkludere alle css-klassene direkte i js-filen, f.eks.
```
<div classNames={`${styles.font} ${styles.box}`}>...</div>
```
(Syntaxen blir enda penere dersom man etterhvert skal gå over til react-css-modules eller babel-plugin-react-css-modules.)
Jeg liker da å ha css-klasser som beskriver hva de gjør (f.eks. font eller background), heller enn å beskrive hvilket element de skal brukes på (som firstElement og leftEdge).

Dette siste bestemmer du selv, hvis du vil beholde "composes", så er det helt greit, så lenge det ikke blir veldig mange kombinasjoner/permutasjoner som man ellers kunne unngått.... :)